ddtrace/settings/integration.py

```diff
@@ -102,6 +103,7 @@ def header_is_traced(self, header_name):
         """
         return self._header_tag_name(header_name) is not None
 
+    @memoize
```

Comment:
Can we use `ddtrace.internal.utils.cache` instead? It seems that the psutil cache has no size bound.